labs/firepower-restapi-101/2.md

```diff
@@ -1,53 +1,59 @@
+
 ## Authentication to the API Explorer
 The REST API relies on the same authentication as the FMC UI. Each function in the REST API maps to permissions in the FMC.
 
-You can log into the API Explorer using any account on the FMC, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the FMC Configuration Guide.
+You can log into the API Explorer using any account on the FMC, but you will only be able to perform the functions for which the account has permissions. For more information on setting permissions for user roles, see the FMC Configuration Guide:
+http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html
+
 
-FMC REST API uses token based authentication. To invoke a REST call on FMC, you need to use a valid access token. On every REST call to access resource URLs you need to add a header with "key" set to “X-auth-access-token” and “value” containing a valid access token . Api explorer also follows this, but it is automated by Explorer UI. You can learn more in firepower-restapi-102 lab.
+FMC REST API uses token based authentication. To invoke a REST call on FMC, you need to use a valid access token. On every REST call to access resource URLs you need to add a header with "key" set to “X-auth-access-token” and “value” containing a valid access token. Api explorer also follows this, but it is automated by Explorer UI. You can learn more in firepower-restapi-102 lab:
+https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1
 
 ## Logging into API Explorer
-The REST API uses the same authentication and permission model as the Firepower Management Center. See the Firepower Management Center Configuration Guide for more information.
+The REST API uses the same authentication and permission model as the Firepower Management Center. See the FMC Configuration Guide for more information.
 http://www.cisco.com/c/en/us/td/docs/security/firepower/610/configuration/guide/fpmc-config-guide-v61/logging_into_firepower_system.html
 
-Make sure you have an account on your Firepower Management Center with the required permissions to perform the tasks you want to perform using the REST API.
+Make sure you have an account on your FMC with the required permissions to perform the tasks you want to perform using the REST API.
 
-## Logging Procedure
-*  Navigate to the following URL: https://management_center_IP_or_name/api/api-explorer or try devnet live sandbox https://fmcrestapisandbox.cisco.com/api/api-explorer/
-*  Log in with your username and password and click **Sign In**
+## Logging procedure
+*  Navigate to the following URL: https://management_center_IP_or_name/api/api-explorer or try https://fmcrestapisandbox.cisco.com/api/api-explorer/ (the devnet live sandbox).
+*  Log in with your username and password and click **Sign In**.
```

Comment:
devnet -> DevNet -- not sure we use the term "live" sandbox, I've not seen that. Maybe Always-On Sandbox?